src/geovista/bridge.py

```diff
@@ -816,11 +818,14 @@ def from_tiff(
             )
             raise ImportError(emsg) from None
 
+        if rgb is None:
+            rgb = False
+
         if isinstance(fname, str):
             fname = Path(fname)
 
         fname = fname.resolve(strict=True)
-        band = None if rgb else band
```

Comment:
MyPy behaved better with consistent types, and as far as I can see `None` and `-1` are irrelevant anyway given the way `rgb` is handled.

Reply:
b172df70ecdccf95223b3cad53667731fedf051d